build.sh

```diff
@@ -0,0 +1,5 @@
+rm -rf build/
```

Comment:
I guess we can remove this file?

Reply:
yeah this is for my own deployment so I could actually use this